hoppfish/hwangsae-user-command.py

```diff
@@ -0,0 +1,78 @@
+#
+#  Copyright 2019 SK Telecom Co., Ltd.
+#    Author: Heekyoung Seo <hkseo@sk.com>
```

Comment:
@wlozano0collabora, you are the author of this file.
Cloud you fix it?

Reply:
Sure.

Reply:
Sure